earthly/rust/Earthfile

```diff
@@ -0,0 +1,75 @@
+# Common Rust UDCs and Builders.
+VERSION 0.7
+
+# cspell: words rustup automake autotools xutils miri nextest kani
+
+# Base Rustup build container.
+rustup:
+    # Base it on Debian bookworm-slim
+    FROM debian:bookworm-slim
+
+    WORKDIR /root
+
+    # Install necessary packages 
+    # Expand this list as needed, rather than adding more tools in later containers.
+    RUN apt-get update && \
+        apt-get install --no-install-recommends -y \
+            ca-certificates \
+            curl \
+            file \
+            build-essential \
+            autoconf \
+            automake \
+            autotools-dev \
+            libtool \
+            xutils-dev \
+            mold \
+            python3 \
```

Comment:
Why do we need Python?

Reply:
The `kani-verifier` needs it as a build dependency.
Rather than scatter `apt-get install` all over the place I figured its best just to do all the dependencies in one place.